src/libraries/System.Net.Quic/tests/FunctionalTests/MsQuicCipherSuitesPolicyTests.cs

```diff
@@ -62,11 +63,14 @@ public void NoSupportedCiphers_ThrowsArgumentException(TlsCipherSuite[] ciphers)
                     return ValueTask.FromResult(serverOptions);
                 }
             };
-            Assert.ThrowsAsync<ArgumentException>(async () => await CreateQuicListener(listenerOptions));
+            await using var listener = await CreateQuicListener(listenerOptions);
 
-            var clientOptions = CreateQuicClientOptions(new IPEndPoint(IPAddress.Loopback, 5000));
+            var clientOptions = CreateQuicClientOptions(listener.LocalEndPoint);
             clientOptions.ClientAuthenticationOptions.CipherSuitesPolicy = policy;
-            Assert.ThrowsAsync<ArgumentException>(async () => await CreateQuicConnection(clientOptions));
+            await Assert.ThrowsAsync<ArgumentException>(async () => await CreateQuicConnection(clientOptions));
+
+            await Assert.ThrowsAsync<AuthenticationException>(async () => await CreateQuicConnection(listener.LocalEndPoint));
+            await Assert.ThrowsAsync<ArgumentException>(async () => await listener.AcceptConnectionAsync());
```

Comment:
comments for individual cases could help readability
```suggestion
            // Creating a connection with incompatible ciphers
            await Assert.ThrowsAsync<ArgumentException>(async () => await CreateQuicConnection(clientOptions));

            // Creating a connection to a server configured with incompatible ciphers
            await Assert.ThrowsAsync<AuthenticationException>(async () => await CreateQuicConnection(listener.LocalEndPoint));
            await Assert.ThrowsAsync<ArgumentException>(async () => await listener.AcceptConnectionAsync());
```

Reply:
Fixed